weed/shell/command_volume_fsck.go

```diff
@@ -209,7 +231,7 @@ func (c *commandVolumeFsck) collectFilerFileIdAndPaths(dataNodeVolumeIdToVInfo m
 			}
 			dataChunks = append(dataChunks, manifestChunks...)
 			for _, chunk := range dataChunks {
-				if chunk.Mtime > collectMtime {
+				if collectMtime != 0 && chunk.Mtime > collectMtime {
```

Comment:
collectCutoffTsNs

`chunk.Mtime > collectMtime` is not valid

Reply:
> collectCutoffTsNs
> 
> `chunk.Mtime > collectMtime` is not valid
> 
> chunk.Mtime is in seconds

```				fmt.Fprintf(c.writer, "chunk.Mtime %d > collectMtime %d ?\n", chunk.Mtime, collectMtime)
```
I empirically found out that this is nanoseconds
```
chunk.Mtime 1666780116556794000 > collectMtime 1666934113759767000 ?
chunk.Mtime 1666780118790098000 > collectMtime 1666934113759767000 ?
chunk.Mtime 1666780138851018000 > collectMtime 1666934113759767000 ?
chunk.Mtime 1666780307287030000 > collectMtime 1666934113759767000 ?
chunk.Mtime 1666780307347438000 > collectMtime 1666934113759767000 ?

```

Reply:
> collectCutoffTsNs
> 
> `chunk.Mtime > collectMtime` is not valid
> 
> chunk.Mtime is in seconds

@chrislusf  I empirically found out that this is nanoseconds.
```
fmt.Fprintf(c.writer, "chunk.Mtime %d > collectMtime %d ?\n", chunk.Mtime, collectMtime)
log:
chunk.Mtime 1666934096263068000 > collectMtime 1666934113759767000 ?
chunk.Mtime 1666934091058753000 > collectMtime 1666934113759767000 ?
chunk.Mtime 1666934096339343000 > collectMtime 1666934113759767000 ?
chunk.Mtime 1666934096713572000 > collectMtime 1666934113759767000 ?
chunk.Mtime 1666934096718220000 > collectMtime 1666934113759767000 ?
```

Reply:
You are right. I refactored the variable just now to ModfiedTsNs. Please sync.

Reply:
ok